docs/resources/resource.md

```diff
@@ -64,6 +75,15 @@ resource "twingate_resource" "resource" {
 
 - `id` (String) Autogenerated ID of the Resource, encoded in base64
 
+<a id="nestedblock--access"></a>
+### Nested Schema for `access`
+
+Optional:
+
+- `group_ids` (Set of String) List of Group IDs that have permission to access the Resource.
+- `service_account_ids` (Set of String) List of Service Account IDs that have permission to access the Resource.
```

Comment:
--> "List of Service Account IDs that must have permission to access the Resource."